packages/@okta/vuepress-site/docs/guides/pwd-optional-new-sign-up-email/main/index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Sign up for new account with email only
+---
+
+<div class="oie-embedded-sdk">
+
+<ApiLifecycle access="ie" /><br>
+
+Learn how to use the Embedded SDK to integrate password optional sign up in your app.
+
+**Learning outcomes**
+
+* Configure your Okta org to allow your users the option to sign up without a password
+* Enable your app to support password optional sign ups
```

Comment:
```suggestion
* Enable your app to support password-optional sign-up flows.
```

Reply:
Issue fixed in PR: https://github.com/okta/okta-developer-docs/pull/3435